docs/dg/dev/backend-development/data-manipulation/data-publishing/publish-and-synchronization.md

```diff
@@ -177,3 +177,36 @@ When a change happens in the mirror table, its *synchronization behavior* sends
 	}
 }
 ```
+
+### Data Architecture 
+
+P&S plays a major role in data denormalization and distribution to Spryker storefronts and API. Denormalization procedure aims for preparing data in the form it will be consumed by data clients. Distribution procedure aims to distribute the data closer to the end users, so that they feel like accessing a local storage.
```

Comment:
```suggestion
P&S plays a major role in data denormalization and distribution to Spryker storefronts and API.
- Denormalization procedure aims for transforming data to the format that will be consumed by data clients.
- Distribution procedure aims to distribute data, so the users feel like accessing a local storage.
```

Reply:
I want to keep it as a paragraph. Not everything should be a bulletpoint)